modules/phase_field/doc/content/source/materials/GBEvolution.md

```diff
@@ -1 +1,34 @@
-!template load file=stubs/moose_object.md.template name=GBEvolution syntax=/Materials/GBEvolution
\ No newline at end of file
+# GBEvolution
+
+!syntax description /Materials/GBEvolution
+
+## Overview
+
+This material calculates the order parameter mobility $L$, the free energy prefactor $\mu$, and the gradient multiplier $\kappa$ for the grain growth model from [!cite](moelans_quantitative_2008), assuming isotropic grain boundary (GB) properties. The parameters are calculated based on the GB energy $\gamma$, the GB mobility $M_{GB}$, and the phase field interfacial width $w_{GB}$ according to
+
+\begin{equation}
+L = \frac{4}{3} \frac{M_{GB}}{w_{GB}}
+\end{equation}
+\begin{equation}
+\mu = \frac{3}{2} \frac{\gamma}{w_{GB}}
```

Comment:
please see previous comment about value of the prefactor and switching from \gamma to \sigma for interfacial energy